HACKING.md

```diff
@@ -42,15 +42,31 @@ Then update the operator ServiceAccount in the hypershift namespace:
 
 ### How to run the e2e tests
 
-1. Install HyperShift.
+1. Complete [Prerequisites](https://hypershift-docs.netlify.app/getting-started/#prerequisites) with a public Route53
+Hosted Zone
+
+2. Install the HyperShift Operator on a cluster, filling in variables such as the S3 bucket name and region based on
+what was done in the prerequisites phase and potentially supplying a custom image.
+
+   ```shell
+   bin/hypershift install \
+   --oidc-storage-provider-s3-bucket-name "${BUCKET_NAME}" \
+   --oidc-storage-provider-s3-credentials "${AWS_CREDS}" \
+   --oidc-storage-provider-s3-region "${AWS_REGION}" \
+   --hypershift-image quay.io/my/hypershift:latest
```

Comment:
Maybe change the hardcoded image to be like an environment variable as well to keep it consistent with the other variables?